Modules/_struct.c

```diff
@@ -97,10 +92,8 @@ typedef struct { char c; _Bool x; } st_bool;
 #define LONG_ALIGN (sizeof(st_long) - sizeof(long))
 #define FLOAT_ALIGN (sizeof(st_float) - sizeof(float))
 #define DOUBLE_ALIGN (sizeof(st_double) - sizeof(double))
-#ifdef Py_HAVE_C_COMPLEX
-#  define FLOAT_COMPLEX_ALIGN (sizeof(st_float_complex) - sizeof(float complex))
-#  define DOUBLE_COMPLEX_ALIGN (sizeof(st_double_complex) - sizeof(double complex))
-#endif
+#define FLOAT_COMPLEX_ALIGN (sizeof(st_float_complex) - 2*sizeof(float))
```

Comment:
I'm wondering whether we really need this define, or the `st_float/double_complex` structs. The alignment of a complex is the same as the alignment of the real type.

Reply:
Yes, these structures are pre-C11 remnants.

In principle, we could use `_Alignof` operator to query alignment for types.  c11-compatible compiler is a build requirement.  Issue: https://github.com/python/cpython/issues/132868